src/ApiConnectors/InvoiceApiConnector.php

```diff
@@ -73,7 +77,11 @@ public function sendAll(array $invoices): void
                 $invoicesDocument->addInvoice($invoice);
             }
 
-            $this->sendDocument($invoicesDocument);
+            $response = $this->sendDocument($invoicesDocument);
```

Comment:
Did you test if this works correctly? 

The invoices are sent in chunks (of 250 objects). However, you call the `InvoiceMapper::map()` only once per sent document. 

Can you test sending multiple invoices and getting multiple mapped invoices back?

Reply:
Hello Willem,

I will run some tests today / tomorrow and get back you :)